src/_gettsim/arbeitslosengeld_2/regelbedarf.py

```diff
@@ -96,47 +225,29 @@ def mehrbedarf_alleinerziehend_m(
 def kindersatz_m_bis_2010(
```

Comment:
```suggestion
def kindersatz_m_anteilsbasiert(
```
Reason is that the `ab_2011` does not make sense when describing a 2022 change. Please change those two accordingly, adding `_nach_regelbedarfsstufen`